public/styles/common.css

```diff
@@ -64,6 +64,15 @@ p.error, li.error, span.error {
 	color: red;
 }
 
+p > span.repoError {
+	color: red;
+	display: none;
+}
+
+p > input:invalid + span.repoError, p > span.repoError.visible {
```

Comment:
:+1:

Reply:
BTW do we really need `p > span.repoError.visible` ? (looks like a relict from an older version)

Reply:
I would keep it for compatibility, not really needed but idk I think it's nice to have

Reply:
For what compatibility? CSS3 is assumed everywhere else and you just introduced `repoError` in this PR.

Reply:
https://caniuse.com/#feat=form-validation